contribs/gmf/apps/mobile_alt/js/controller.js

```diff
@@ -58,6 +58,32 @@ app.AlternativeMobileController = function($scope, $injector) {
    */
   this.searchCoordinatesProjections = ['EPSG:21781', 'EPSG:2056', 'EPSG:4326'];
 
+
+  /**
+   * @type {ol.style.Style}
+   * @export
+   */
+  this.customMeasureStyle = new ol.style.Style({
```

Comment:
please add the corresponding `goog.require`

Reply:
Added it.